server/middleware/form/validation.js

```diff
@@ -605,9 +575,60 @@ function validationMiddleware(req, res, next) {
             }
         }
     }
+
+    /**
+     * Check to see if field component is of display only (without validation rules).
+     * @param field the field to check
+     * @returns {boolean} whether the field should be kept (true)
+     */
+    function isDisplayComponent(field) {
```

Comment:
We also have this logic in process.js on line 86, potential to share across with something like isComponentOfType(type) instead maybe?

```
return field.component === type
```

Reply:
I've made a change - good shout - think I've done it as you suggested.